packages/account/src/Components/currency-selector/radio-button.tsx

```diff
@@ -1,11 +1,24 @@
-import React from 'react';
+import React, { InputHTMLAttributes, AllHTMLAttributes, ReactElement } from 'react';
 import classNames from 'classnames';
 import { Localize } from '@deriv/translations';
 import { Popover, Icon } from '@deriv/components';
 import { getCurrencyDisplayCode } from '@deriv/shared';
 
-const USTPopover = ({ id }) => {
-    let popover_message;
+type TUSTPopover = {
+    id: string;
+};
+
+type TRadioButtonExtend = {
+    children: never;
```

Comment:
The way I see It, shouldn't this be ReactNode?

Reply:
This component does not accept any children. React implicitly has typed children as optional. To override and warn the user when a child is passed to it, type `never` can be used

Reply:
Thank u for your explanation